### PR TITLE
Jose

### LINKED ISSUE #29: Assurance Case 3

### PR DESCRIPTION
**Pull request by Jbibianochavez**
After our meeting I have made some revisions to the diagram. Please review the content, I am working on resizing the image.

### CODE REVIEW

File: Assurance Cases.md [hunk 1]
```diff
@@ -37,8 +37,7 @@
 ### Assurance Case 3
 #### .red[TODO] Insert Assurance Case 3 Name
 
-![Assurance Case 3 Diagram:](/images/)
-.red[TODO] Insert Assuracne Case 3 Image
+![Assurance Case 3 Diagram:](/images/IoT_Assurance_Case.png)
```
Comment (Chrs987) [inline Assurance Cases.md:53]: For your evidence, remove the verbs. Evidence should be noun phrases only. Evidence E3 should be worded as The python library on Pypi and separate the other one to another evidence piece 'The integration manifest for the device' or something similar. Evidence is not supposed to be stated as a claim or have verbs in it.
Evidence E2 - API logs or something similar 
You have 2 sub-claim C2. 
Undermine UM1: - Wording, remove the word "or"
Also for the subclaims what are the "critical properties" and "values" 
Example: Sub-claim C2 - A user can securely integrate a device to Home Assistant. 
Discovery will effectively discover and configure compatible devices. 
Also Rebuttal R2 - Unless the device is "**incompatible**" not compatible
sub claim C4 - 3rd party* not part.
Doesnt sub-claim C6 need to be closed with evidence or is the diamond fine?

**Review by jmpinoy — DISMISSED** [review]
Everything looks good

**Review by Chrs987 — DISMISSED** [review]
Rebuttal R3 do you mean integrates instead of interfaces?
Rebuttal R2 do you mean incompatible instead of compatible?
Other than that looks good, can you make the image a little smaller?

**Comment by Jbibianochavez** [comment]
All points made in the comments have been addressed and the picture has been resized.

**Review by Chrs987 — DISMISSED** [review]
Changes look good!

**Review by Chrs987 — DISMISSED** [review]
Nice job on the changes! our understanding is in tune now that we are on the same page lol. Thanks!

**Review by Chrs987 — APPROVED** [review]
Much better! I don't know how the image defaulted to that large but it looks smaller now

**Review by Chrs987 — APPROVED** [review]
Changes look good thanks

**Review by RocketSockz — APPROVED** [review]

**Pull request merged**